pipeline/recon/helpers.py

```diff
@@ -19,6 +21,15 @@ def get_tool_state() -> typing.Union[dict, None]:
         return pickle.loads(tools.read_bytes())
 
 
+def meets_requirements(requirements):
+    """ Determine if tools required to perform task are installed. """
+    tools = get_tool_state()
+
+    for tool in requirements:
+        if not tools.get(tool).get("installed"):
+            raise (Exception(style(f"[!] {tool} is not installed", fg="bright_red")))
```

Comment:
- parens surrounding Exception aren't necessary
- RuntimeError is probably a better option for what to raise

consider us calling the function

```
try:
    meets_requirements(['stuff'])
except Exception:
    # handle stuff
```

Literally everything that can possibly go wrong is caught under that, with potentially non-specific exception handling.  Better practice is to not catch the base Exception and be more specific.